### PR TITLE
Adding lucit-licensing-python

### PR DESCRIPTION
**Pull request by oliver-zehentleitner**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by oliver-zehentleitner** [comment]
@conda-forge/staged-recipes, ready for review
@conda-forge-admin, please ping team

**Comment by oliver-zehentleitner** [comment]
@conda-forge/help-python-c

File: recipes/lucit-licensing-python/meta.yaml
Comment (ocefpaf) [inline recipes/lucit-licensing-python/meta.yaml:28]: Is cython really needed at run time?

File: recipes/lucit-licensing-python/meta.yaml
Comment (ocefpaf) [inline recipes/lucit-licensing-python/meta.yaml:47]: Doesn't look like MIT: https://github.com/LUCIT-Systems-and-Development/lucit-licensing-python/blob/main/LICENSE#L1

Comment (oliver-zehentleitner) [reply recipes/lucit-licensing-python/meta.yaml:47]: Thank you! "LSOSL" als license is not accepted. what can i do to solve this?

File: recipes/lucit-licensing-python/meta.yaml
Comment (oliver-zehentleitner) [reply recipes/lucit-licensing-python/meta.yaml:28]: Yes! Its loaded and used in runtime.

File: recipes/lucit-licensing-python/meta.yaml
Comment (ocefpaf) [reply recipes/lucit-licensing-python/meta.yaml:47]: What do you mean by not accepted?

Comment (oliver-zehentleitner) [reply recipes/lucit-licensing-python/meta.yaml:47]: If i replace MIT with LSOSL in the recipe I get this hint:

> For recipes/lucit-licensing-python:
> 
>     License is not an SPDX identifier (or a custom LicenseRef) nor an SPDX license expression.

This just offers a limited list of open source licenses where mine is not inluded.

Comment (oliver-zehentleitner) [reply recipes/lucit-licensing-python/meta.yaml:47]: ah, thanks, did not read correctly! I will try again. Yes i am sure the license fits, its my own project and i wrote the license myself.

File: recipes/lucit-licensing-python/meta.yaml
Comment (oliver-zehentleitner) [reply recipes/lucit-licensing-python/meta.yaml:28]: https://github.com/LUCIT-Systems-and-Development/lucit-licensing-python/blob/8061f780547743c820631987652f17a8c8642857/lucit_licensing_python/manager.py#L19

File: recipes/lucit-licensing-python/meta.yaml
Comment (oliver-zehentleitner) [reply recipes/lucit-licensing-python/meta.yaml]: Ok, linter accepted it and other checks passed also. Thank you very much :D

**Comment by oliver-zehentleitner** [comment]
@conda-forge-admin, please ping team

File: recipes/lucit-licensing-python/meta.yaml
Comment (ocefpaf) [reply recipes/lucit-licensing-python/meta.yaml]: @xhochy do you mind taking a look at this license before we merge? I missed important details in the last one.

@oliver-zehentleitner consider using an OSI license for your projects. It makes the package adoption and distribution much easier.

Comment (oliver-zehentleitner) [reply recipes/lucit-licensing-python/meta.yaml]: I think switching is not possible, OSI says that we are not allowed to charge a license fee. but we need to do that.

Comment (oliver-zehentleitner) [reply recipes/lucit-licensing-python/meta.yaml]: Which important detail do you miss in the license i use?

Comment (ocefpaf) [reply recipes/lucit-licensing-python/meta.yaml]: Well, if I knew I wouldn't be missing it :-)

I just want another set of eyes here b/c I did make a bad merge recently. Just want to avoid another mistake on my part.

Comment (xhochy) [reply recipes/lucit-licensing-python/meta.yaml]: The LICENSE file only refers to a mail address regarding distribution. Overall, the LICENSE file does not state a license but only refers that you need an actual license to use it. I don't think we can ship this.

**Comment by jakirkham** [comment]
Please see this question above: https://github.com/conda-forge/staged-recipes/pull/24449#discussion_r1387796686

**Review by xhochy — DISMISSED** [review]
To make it clear: I don't see a license here that would allow us to distribute this package.

File: recipes/lucit-licensing-python/meta.yaml
Comment (oliver-zehentleitner) [reply recipes/lucit-licensing-python/meta.yaml]: @xhochy I reworked the license. Please review it! Thank you!

Comment (xhochy) [reply recipes/lucit-licensing-python/meta.yaml]: I have no objections anymore but I also don't feel comfortable approving this. I have removed my ❌ though.

File: recipes/lucit-licensing-python/meta.yaml
Comment (BastianZim) [inline recipes/lucit-licensing-python/meta.yaml:13]:
```suggestion
```
This can be removed as conda-forge does not build for python3.6

**Review by BastianZim — CHANGES_REQUESTED** [review]

File: recipes/lucit-licensing-python/meta.yaml
Comment (BastianZim) [reply recipes/lucit-licensing-python/meta.yaml]: I would be fine with distributing the license; my objection is that it does not allow usage without a license. As the software is used in the test, we would need a license that allows us to do so.

I am not a part of core though so just my 2 cents

Comment (oliver-zehentleitner) [reply recipes/lucit-licensing-python/meta.yaml]: @xhochy Thank you!
@BastianZim That's right, this would actually be a license violation by definition. I will extend the license for this and get back to you.

File: recipes/lucit-licensing-python/meta.yaml
Comment (oliver-zehentleitner) [reply recipes/lucit-licensing-python/meta.yaml]: Done, thanks!

File: recipes/lucit-licensing-python/meta.yaml
Comment (oliver-zehentleitner) [reply recipes/lucit-licensing-python/meta.yaml]: I added this to our license:
https://github.com/LUCIT-Systems-and-Development/lucit-licensing-python/blob/8bfd312fe7e59e4cb0457bb7ed6393ee29c650d4/LICENSE#L67

> Furthermore, this license explicitly permits Conda-Forge, an open-source community-driven platform for package 
> management and distribution operated by NumFOCUS, to test the software package following its creation to ensure 
> functionality, compatibility, and security standards are met. This permission is granted to support the community's 
> efforts in maintaining a robust ecosystem for software distribution and to guarantee the quality and performance of  
> the software for end users.
>

Comment (BastianZim) [reply recipes/lucit-licensing-python/meta.yaml]: That looks acceptable to me, but I would like to get a second pair of eyes on this.

@ocefpaf and @xhochy what do you think?

Comment (oliver-zehentleitner) [reply recipes/lucit-licensing-python/meta.yaml]: @conda-forge-admin, please ping team

**Comment by ocefpaf** [comment]
> @ocefpaf and @xhochy what do you think?

I'll abstain from commenting b/c I'm not comfortable with custom licenses. I'm not a lawyer nor an expert on the topic.

**Comment by xhochy** [comment]
> > @ocefpaf and @xhochy what do you think?
> 
> I'll abstain from commenting b/c I'm not comfortable with custom licenses. I'm not a lawyer nor an expert on the topic.

I also abstain from commenting/approving b/c I feel that this custom license requires a lawyer to check.

**Comment by BastianZim** [comment]
@oliver-zehentleitner, it might be easier for you to publish it on your own channel. These packages are complicated for us as we are all volunteers and have no legal support etc.

**Comment by oliver-zehentleitner** [comment]
I am sorry if I am causing you any inconvenience and I thank you for your time.

I don't think it needs to be that complicated. Our license is very short and concise and contains no hidden or confusingly worded statements.

A comprehensive license for Conda-Forge is already integrated. Perhaps we could formulate your identity even more precisely, that wouldn't be bad. Otherwise, you have been granted all the authorizations you need. We would be happy to confirm this again in a separate document and send it to you for your archive.

Is there a way to save "secrets" like in GitHub Actions? Then we could also provide a proper license with which the modules can be started for testing after creation. This is how we do it in the build process for Pypi and GitHub.

**Comment by BastianZim** [comment]
Unfortunately, we do not have a legal entity, and the volunteers here are just that and not acting on behalf of an official entity registered somewhere. Hence, the legal implications are challenging to consider. Since we also don't have any legal support, etc, it is pretty much impossible for us to gauge this.
Saving of secrets is not available, as the infrastructure is not customised per feedstock. If you are still interested in publishing your package on conda, I would highly recommend creating your own channel. You can even include the publishing as part of your CI so that everything would be automated.
Alternatively, I have seen similar situations where the module was licensed under an OpenSource license, and further content was downloaded from a private conda channel or only unlocked when the license was provided. Maybe something else to explore?

**Comment by oliver-zehentleitner** [comment]
I think it's a pity, I think conda-forge is very good and it simplifies a lot for the user because everything comes consistently from one channel. I have now created https://anaconda.org/lucit and am migrating the packages... 

I'm closing this!

**Pull request closed without merging**
